test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -151,12 +151,12 @@ end
                 Ẋ = rand_tangent(X)
                 F = eigen!(copy(X))
                 F_fwd, Ḟ_ad = frule((ZeroTangent(), copy(Ẋ)), eigen!, copy(X))
-                @test F_fwd == F
+                @test F_fwd ≈ F
                 @test Ḟ_ad isa Tangent{typeof(F)}
                 Ḟ_fd = jvp(_fdm, asnt ∘ eigen! ∘ copy, (X, Ẋ))
                 @test Ḟ_ad.values ≈ Ḟ_fd.values
                 @test Ḟ_ad.vectors ≈ Ḟ_fd.vectors
-                @test frule((ZeroTangent(), ZeroTangent()), eigen!, copy(X)) == (F, ZeroTangent())
+                @test frule((ZeroTangent(), ZeroTangent()), eigen!, copy(X)) ≈ (F, ZeroTangent())
```

Comment:
`≈` doesn't work on tuples.
And is deprecated on `AbstractTangents`

I think instead use the `test_approx` from ChainRulesTestUtils.
Similar above.

```suggestion
                test_approx(frule((ZeroTangent(), ZeroTangent()), eigen!, copy(X)), (F, ZeroTangent()))
```